.github/workflows/component-template-e2e-tests.yml

```diff
@@ -33,24 +33,8 @@ jobs:
         with:
           python-version: "${{ env.PYTHON_MAX_VERSION }}"
 
-      - name: Setup virtual env
-        uses: ./.github/actions/make_init
-
-      - name: Run make develop
-        run: make develop
-
-      - name: Build Package
-        timeout-minutes: 120
-        run: BUILD_AS_FAST_AS_POSSIBLE=1 make package
-
-      - name: Return path to Streamlit wheel
-        id: streamlit_wheel
-        shell: bash
-        run: |
-          streamlit_wheel=$(find "./lib/dist" -maxdepth 1 -name '*.whl')
-          streamlit_wheel=$(readlink -e "${streamlit_wheel}")
-          echo "output_file=${streamlit_wheel}" >> $GITHUB_OUTPUT
-
+      # Build the component lib first so we can use the old version of yarn
```

Comment:
Out of curiosity, do you know what's the delta for using the new yarn version also in the component-lib module? If it is not big, I think it'd be worth it to do the update simultaneously instead of fragmenting the versions